src/commands/app/doctor.ts

```diff
@@ -0,0 +1,240 @@
+import { flags } from "@oclif/command";
+
+import { Kommand } from "../../common";
+import { kuzzleFlags } from "../../support/kuzzle";
+import { Client } from "@elastic/elasticsearch";
+import { execute } from "../../support/execute";
+import _ from "lodash";
+
+export default class AppDoctor extends Kommand {
+  static description = "Analyze a Kuzzle application";
+
+  static flags = {
+    help: flags.help(),
+    elasticsearch: flags.string({
+      description: "Elasticsearch server URL",
+      default: "http://localhost:9200",
+    }),
+    ...kuzzleFlags,
+  };
+
+  static args = [];
+
+  static readStdin = true;
+
+  ELK_MAX_VERSION = "7.17";
+  NODEJS_MAX_VERSION = "14";
+
+  COMMANDS_PLATFORMS: { [key: string]: string } = {
```

Comment:
Since this is not modified, it should be defined as const without explicit type and then TS will automatically use keys present inside as type definition

https://github.com/kuzzleio/kuzzle-device-manager/blob/2-stable/features/fixtures/application/decoders/DummyTempDecoder.ts#L11

Reply:
I try that, but I have this compilation problem:

TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ readonly linux: "which"; }'.   No index signature with a parameter of type 'string' was found on type '{ readonly linux: "which"; }'.

Reply:
Thanks ocliff <3 Can you add a comment explaining why then?